extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -165,14 +165,15 @@ export class AmpSidebar extends AMP.BaseElement {
     });
 
     // Get the toolbar attribute from the child navs.
-    const toolbarElements = toArray(element.querySelectorAll('nav[toolbar]'));
-
-    toolbarElements.forEach(toolbarElement => {
-      try {
-        this.toolbars_.push(new Toolbar(toolbarElement, this));
-      } catch (e) {
-        this.user().error(TAG, 'Failed to instantiate toolbar', e);
-      }
+    this.getAmpDoc().whenReady(() => {
```

Comment:
`this.getAmpDoc().whenReady().then(() => {})`?

Reply:
Oops, thanks. :)